test/parallel/test-performance-function.js

```diff
@@ -43,27 +43,6 @@ const {
   obs.disconnect();
 }
 
-{
-  class N {}
-  const n = performance.timerify(N);
-
-  const obs = new PerformanceObserver(common.mustCall((list) => {
-    const entries = list.getEntries();
-    const entry = entries[0];
-    assert.strictEqual(entry[0], 1);
-    assert.strictEqual(entry[1], 'abc');
-    assert(entry);
-    assert.strictEqual(entry.name, 'N');
-    assert.strictEqual(entry.entryType, 'function');
-    assert.strictEqual(typeof entry.duration, 'number');
-    assert.strictEqual(typeof entry.startTime, 'number');
-    obs.disconnect();
-  }));
-  obs.observe({ entryTypes: ['function'] });
-
-  new n(1, 'abc');
-}
-
```

Comment:
unrelated changes?

Reply:
I've removed this test due to it being no longer functional as I've changed the behavior of the `validateFunction()` validator to no longer accept classes as a function prior to the referenced PR.